scss/backend/dot-admin/portlets/_content-edit.scss

```diff
@@ -79,4 +79,27 @@ $sidebar-width: 220px;
       height: 0;
       z-index: 1;
     }
+}
+
+.content-edit__history-version .dijitContentPane {
+  padding: 0px;
+
+  .contentIdentifier {
+    font-weight: bold;
+  }
+}
+
+.content-edit__divider {
+  background-color: $gray-ligther;
+  margin: 20px 0;
+}
+
+.content-edit__cell-aling-center {
```

Comment:
We can't have class name that refer properties, like: ```align-center``` or ```margin-big``` or ```font-big``` we need the class names semantic, more info: http://maintainablecss.com/chapters/semantics/